modes/vterm/evil-collection-vterm.el

```diff
@@ -91,6 +91,11 @@ also uses `evil-mode'."
   (vterm-goto-char (vterm--get-end-of-line))
   (call-interactively #'evil-append))
 
+(defun evil-collection-vterm-paste-after (&optional arg)
```

Comment:
`arg` -> `_arg`

Reply:
Thank you for the review.
I used it for `vterm-yank` like so

```
 (call-interactively #'vterm-yank arg))
```

I am not sure if it is useful but I think it's better than just ignoring the parameter.

Reply:
OK